api/pom.xml

```diff
@@ -13,6 +13,7 @@
     <properties>
         <project.root>${basedir}/..</project.root>
         <springfox.version>2.9.2</springfox.version>
+        <springboot.version>2.1.6.RELEASE</springboot.version>
```

Comment:
@adaykin Let's declare this property in the top level `pom`, since that is where the `spring-boot-starter-parent` is declared. The property will propagate to the modules automatically.

Reply:
Ok. Would it be excluded from the domain module since I don't believe it would be needed there?

Reply:
Actually I guess this is just the property so it's fine.